src/Zypper.cc

```diff
@@ -5745,6 +5835,41 @@ void Zypper::doCommand()
     break;
   }
 
+  case ZypperCommand::KEYS_e:
```

Comment:
```
$ zypper keys 39DB7C82
+-------------------+
|   . . ^           |  Key ID:           70AF9E8139DB7C82
|    : : .          |  Key Name:         SuSE Package Signing Key <build@suse.de>
...
$ zypper keys 9DB7C82
Warning: Argument '9DB7C82' is not a trusted ID nor a existing file.
$ zypper keys @suse.de
Warning: Argument '@suse.de' is not a trusted ID nor a existing file.
+----[39DB7C82]-----+
```
What do you think about relaxing the required argument format? 

Maybe is 'trailing portion of an `ID`' or 'is substring of the `Name`' (all case insensitive).

Same could be done for `removekey`, with the additional constraint, that every arg must match exactly 1 key.  If an arg matches more one could say `"ARG" specifies multiple  keys' (opt. indented list of maches 'ID Name').
`Use --all-matches to remove all metchein keys`.

Reply:
Done